examples/closure-app/silabs/src/DataModelCallbacks.cpp

```diff
@@ -53,5 +55,7 @@ void MatterPostAttributeChangeCallback(const app::ConcreteAttributePath & attrib
 void MatterClosureControlClusterServerAttributeChangedCallback(const app::ConcreteAttributePath & attributePath)
 {
     ChipLogProgress(Zcl, "Closure cluster ID: " ChipLogFormatMEI, ChipLogValueMEI(attributePath.mAttributeId));
-    // Attribute changes handling will done in next phase.
-}
+    if(attributePath.mClusterId == app::Clusters::ClosureControl::Id) {
```

Comment:
This is the `MatterClosureControlClusterServerAttributeChangedCallback`. it shouldn't be called by any other cluster.
I'd assume that if it is, it's an error.
A  `VerifyOrDie` should probably be used